src/charm.py

```diff
@@ -315,6 +305,12 @@ def _handle_status_update_config(self, event: HookEvent) -> None:
             self.unit.status = BlockedStatus("Missing required relation with openfga")
             return
 
+        if not self._validate_cert_trust():
+            self.unit.status = BlockedStatus(
+                "Missing certificate_transfer relation with oauth provider"
+            )
```

Comment:
Do we need to stop the service?

Reply:
makes sense to avoid getting a tons of error logs

Reply:
actually we should do this probably for all our mandatory relations